lib/transition-config/required_homepage_protocol_exception.rb

```diff
@@ -0,0 +1,13 @@
+module TransitionConfig
+  class RequiredHomepageProtocolException < RuntimeError
+    attr_reader :missing
+    def initialize(missing)
+      @missing = missing
+    end
+
+    def to_s
```

Comment:
Multi line concatenation without using `+`. Only to stay consistent with the other Exception Handlers in this repo to be honest.